gen3/bin/kube-dev-namespace.sh

```diff
@@ -92,7 +88,10 @@ mkdir -p credentials
 for name in ca.pem ca-key.pem; do
   cp ~/${vpc_name}/credentials/$name credentials/
 done
-sed -i.bak "s/default/$namespace/" kubeconfig
+
+cp kubeconfig kubeconfig.bak
+rm kubeconfig
```

Comment:
If you are overwriting the file later, there is no need to delete it, but no big deal.